### DISCUSSION TITLE
Screen flickering issue when changing refresh rate

### DISCUSSION THREAD

**Discussion by Biziame** [seerge/g-helper#2162] [Q&A]
I'm currently using your software to manage the refresh rate of my screen. I own an Asus Zephyrus G14 2022 with an RX 6700S graphics and an AMD Ryzen 9 6900HS processor. My graphics driver is Adrenalin 24.1.1.

I've noticed a peculiar behavior. Every time I switch between refresh rates (from 60Hz to 120Hz or vice versa), there's a momentary flicker where the screen briefly turns on and off. However, since updating my graphics card drivers, this flickering no longer occurs, even though the refresh rate change still takes effect.

On the other hand, if I make the same refresh rate change using the default Windows system settings, the screen continues to flicker as it did before.

I'm curious to know if this flickering behavior is normal or if it's something that should be addressed. Is it intended for the screen to briefly turn on and off during refresh rate changes, or is the current behavior acceptable?

**Comment by seerge** [comment]
@Biziame this is something that recent AMD drivers has changed, they have separated display / signal refresh rates.

G-Helper uses the only available official windows API to set refresh rate (which is pretty much a shortcut to corresponding setting), how it happens under the hood - is up to that drivers and windows itself :)

**Reply by Biziame** [reply]
@seerge 
Got it, I understand that the refresh rate change from 120Hz to 60Hz when switching from cable to battery power depends on the behavior of the latest driver update. However, I'm not sure if this modification without the usual screen flickering is something to worry about. Is everything normal?

**Reply by seerge** [reply]
It's same for my G14 2022 :) I don't worry about it.
